### PR TITLE
Create player skill rank migration

### PR DESCRIPTION
**Pull request by chair28980**
Create migration for player skill rank addressing https://github.com/MetaFam/TheGame/issues/441

### CODE REVIEW

**Comment by chair28980** [comment]
Note to self: make a note about the .env.local file pointing to production data.

File: packages/web/pages/players.tsx [hunk 1]
```diff
@@ -15,7 +15,10 @@ import React, { useEffect, useMemo, useRef } from 'react';
 
 type Props = InferGetStaticPropsType<typeof getStaticProps>;
 
+
+
 export const getStaticProps = async () => {
+  console.log("WE MADE IT")
```
Comment (alalonde) [inline packages/web/pages/players.tsx:21]: Oops :D 

No changes of significance, let's just revert back to the version in development

File: packages/web/components/Setup/SetupUsername.tsx [hunk 2]
```diff
@@ -59,7 +59,7 @@ export const SetupUsername: React.FC<SetupUsernameProps> = ({
       <Input
         background="dark"
         placeholder="USERNAME"
-        value={username}
+        value={username ?? ''}
```
Comment (alalonde) [inline packages/web/components/Setup/SetupUsername.tsx:62]: why this change?

**Review by alalonde — COMMENTED** [review]
Sweet! Is this intended to be the backend-only changes for this particular feature?

**Comment by alalonde** [comment]
Ah, I see it's WIP, carry on ;)

File: packages/web/components/Setup/SetupUsername.tsx (hunk 2 above)
Comment (dysbulic) [reply packages/web/components/Setup/SetupUsername.tsx:62]: Going from an undefined variable to a value causes React to complain about switching from an uncontrolled component to a controlled one.

**Comment by richteri** [comment]
Closed in favor of https://github.com/MetaFam/TheGame/pull/807
